resources/build/trigger-builds.inc.sh

```diff
@@ -120,3 +124,39 @@ function triggerJenkinsBuild() {
     echo
   fi
 }
+
+function triggerGitHubActionsBuild() {
+  local IS_TEST_BUILD="$1"
+  local GITHUB_ACTION="$2"
+  local GIT_BRANCH="${3:-master}"
+  local GIT_REF=$GIT_BRANCH
+
+  local GITHUB_SERVER=https://api.github.com/repos/keymanapp/keyman/dispatches
+
+  # This will only be true if we created and pushed a tag
+  if [ "${action:-""}" == "commit" ]; then
+    GIT_REF="release-$VERSION_WITH_TAG"
```

Comment:
```suggestion
    GIT_REF="release@$VERSION_WITH_TAG"
```

We've moved to `release@tag` now.